.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -293,6 +294,8 @@ if [ "${WORKER}" == 0 ]; then
     chmod  770                                      $SUBMITTY_DATA_DIR/to_be_graded_queue
     chown  ${DAEMON_USER}:${DAEMONPHP_GROUP}        $SUBMITTY_DATA_DIR/daemon_job_queue
     chmod  770                                      $SUBMITTY_DATA_DIR/daemon_job_queue
+    chown  ${DAEMON_USER}:${DAEMONPHP_GROUP}        $SUBMITTY_DATA_DIR/grading
+    chmod  770                                      $SUBMITTY_DATA_DIR/grading
```

Comment:
If `submitty_php` is only reading out of this directory, and never writing anything into it, let's make the permission `750`.